mesa/visualization/modules/PyPlotVisualization.py

```diff
@@ -0,0 +1,42 @@
+from mesa.visualization.ModularVisualization import VisualizationElement
```

Comment:
You should sort the import order based on PEP 8: https://www.python.org/dev/peps/pep-0008/#imports.

```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```

Reply:
Updated the code